libs/vkd3d/device.c

```diff
@@ -1596,6 +1597,12 @@ static void vkd3d_physical_device_info_init(struct vkd3d_physical_device_info *i
         vk_prepend_struct(&info->features2, &info->extended_dynamic_state2_features);
     }
 
+    if (vulkan_info->EXT_extended_dynamic_state3)
+    {
+        info->extended_dynamic_state3_features.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_EXTENDED_DYNAMIC_STATE_3_FEATURES_EXT;
+        vk_prepend_struct(&info->features2, &info->extended_dynamic_state3_features);
```

Comment:
Should we explicitly disable all the features we don't use? Not sure if drivers would do anything differently based on those feature bits, but might good to be explicit about which features we actually use. We do similarly for EDS2.

Reply:
Fair point, disabled all features now that aren't used by vkd3d-proton or dxvk.